Oxide.Core/Libraries/WebRequests.cs

```diff
@@ -395,6 +636,25 @@ public void Enqueue(string url, string body, Action<int, string> callback, Plugi
             workevent.Set();
         }
 
+
+        /// <summary>
+        /// Enqueues a DELETE, GET, PATCH, POST, HEAD, or PUT web request
+        /// </summary>
+        /// <param name="url"></param>
+        /// <param name="body"></param>
+        /// <param name="callback"></param>
+        /// <param name="owner"></param>
+        /// <param name="method"></param>
+        /// <param name="headers"></param>
+        /// <param name="timeout"></param>
+        [LibraryFunction("EnqueueV2")]
```

Comment:
`EnqueueV2` looks out of pattern. Probably need to use another naming for new methods/parameters before deprecating others.

Reply:
Library functions don't support same name. These changes are temporary and will be changed later most likely with the release of uMod. Right now this is just a band-aid so we arn't breaking every plugin that uses a webrequest.